easybill-rest-client.gemspec

```diff
@@ -1,32 +1,30 @@
 # -*- encoding: utf-8 -*-
-$:.push File.expand_path("../lib", __FILE__)
-require "easybill-rest-client/version"
+# frozen_string_literal: true
+$LOAD_PATH.push File.expand_path('../lib', __FILE__)
+require 'easybill-rest-client/version'
 
 Gem::Specification.new do |s|
-  s.name        = "easybill-rest-client"
+  s.name        = 'easybill-rest-client'
   s.version     = EasybillRestClient::VERSION
   s.platform    = Gem::Platform::RUBY
-  s.authors     = ["Zeke Sikelianos", "Tony Tam"]
-  s.email       = ["zeke@wordnik.com", "fehguy@gmail.com"]
-  s.homepage    = "http://swagger.io"
-  s.summary     = %q{A ruby wrapper for the swagger APIs}
-  s.description = %q{This gem maps to a swagger API}
-  s.license     = "Apache-2.0"
+  s.authors     = ['ad2games GmbH']
+  s.email       = ['developers@ad2games.com']
+  s.homepage    = 'http://ad2games.com'
+  s.summary     = 'A ruby wrapper for the Easybill REST API'
+  s.description = 'A ruby wrapper for the Easybill REST API'
+  s.license     = 'MIT'
 
-  s.add_runtime_dependency 'typhoeus', '~> 0.2', '>= 0.2.1'
-  s.add_runtime_dependency 'json', '~> 1.4', '>= 1.4.6'
-  s.add_runtime_dependency 'retryable'
+  s.add_runtime_dependency 'faraday', '~> 0.9'
```

Comment:
Influenced by http://www.mikeperham.com/2016/02/09/kill-your-dependencies/ I would switch it to the standard `net/http`. Not in this PR, but just saying. WDYT?